models/models.js

```diff
@@ -68,6 +68,21 @@ exports.updateNewComment = (newComment, id) => {
     .then(({ rows }) => {
         return rows[0]
     })
+}
 
-
+exports.updateArticleVotes = ({ inc_votes }, id) => {
+    if (inc_votes === undefined) {
```

Comment:
Generally it's best to allow psql to throw an error if it's going to and only implement custom error handling if we have to, if we pass these invalid requests to psql what happens? Do we get an error code that we could handle with our middleware?